live-build/misc/live-build-hooks/90-raw-disk-image.binary

```diff
@@ -152,22 +152,17 @@ rsync --info=stats3 -Wa binary/* "$DIRECTORY/"
 # need to have the /dev, /proc, and /sys mountpoints present in that chroot
 # environment, which is why we bind mount here.
 #
-mount --rbind /dev "$DIRECTORY/dev"
-mount --rbind /proc "$DIRECTORY/proc"
-mount --rbind /sys "$DIRECTORY/sys"
+for dir in /dev /proc /sys; do
```

Comment:
Before doing any of that, you'll probably want to call: `mount --make-slave ${DIRECTORY}`, to avoid running into https://jira.delphix.com/browse/LX-1199.